build.gradle

```diff
@@ -197,7 +197,7 @@ tasks.withType(Test) {
 }
 
 test {
-  failFast = true
+  failFast = false
```

Comment:
I'm not certain why we changed it to false.
Are we confident about this change?

Reply:
ah i forgot, i was meant to remove this, thanks for the catch!